.github/actions/test_unit/action.yml

```diff
@@ -34,7 +34,7 @@ runs:
       uses: actions-rs/cargo@v1
       with:
         command: test
-        args: --all-features --no-fail-fast
```

Comment:
Removed `--all-features` here is for?

Reply:
previously, the unit tests of s3  are all attributed as `ignored` (in file `common/dal/tests/it/accessors/aws_s3.rs`), so that these unit tests will be ignored during CI; for those who want to test these cases locally, for example by using a testing aws account or a mock service like minio, the `tests/it/accessors/aws_s3.rs` have to be modified locally -- remove the `ignore` annotations, and re-compile them. that is not that convenient.

so,  a feature `ut_mock_s3` is introduced; if it is not configured, unit tests of s3's will not be enabled. Role of dev could fire up the s3 specific unit tests by using commands like `cargo test --features ut_mock_s3 -p common-dal --test it`, without modifying and re-compile the sources.

unfortunately, if `--all-features` is passed to `cargo test` in CI, s3 unit tests will be enabled, and since there are no corresponding configuration/settings, those cases will always be failed. 

thus, the flag `--all-features` is removed from CI script.